plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/Conversions.java

```diff
@@ -64,8 +64,13 @@ public static ColumnMetadata toColumnMetadata(Field field)
                 .build();
     }
 
-    public static boolean isSupportedType(LegacySQLTypeName type)
+    public static boolean isSupportedType(Field field)
     {
+        LegacySQLTypeName type = field.getType();
+        // Skip BIGNUMERIC without parameters because the precision (77) and scale (38) is too large
+        if (type == LegacySQLTypeName.BIGNUMERIC && field.getPrecision() == null && field.getScale() == null) {
```

Comment:
Should we also skip when p > 38?

Or introduce the `DecimalModule` to allow mapping larger decimals using `decimal-mapping` config (see doc at https://trino.io/docs/current/connector/postgresql.html#decimal-type-handling)?

Reply:
Thanks for catching that. I misunderstood the spec of BIGNUMERIC type. 
Filed #9923. I will add a logic to skip when `p > 38` in this PR.